Scripts/Tracking/Follow/Modifier/FollowModifier.cs

```diff
@@ -50,16 +50,66 @@ public ProcessTarget ProcessType
-        public abstract void UpdatePosition(Transform source, Transform target);
+        public virtual void UpdatePosition(Transform source, Transform target)
+        {
+            if (isActiveAndEnabled && ValidCache(source, target))
+            {
+                DoUpdatePosition(source, target);
+            }
+        }
+
         /// Updates the source rotation based on the target rotation.
         /// </summary>
         /// <param name="source">The source to modify.</param>
         /// <param name="target">The target to utilize in the modification.</param>
-        public abstract void UpdateRotation(Transform source, Transform target);
+        public virtual void UpdateRotation(Transform source, Transform target)
+        {
+            if (isActiveAndEnabled && ValidCache(source, target))
+            {
+                DoUpdateRotation(source, target);
+            }
+        }
+
         /// Updates the source scale based on the target scale.
         /// </summary>
         /// <param name="source">The source to modify.</param>
         /// <param name="target">The target to utilize in the modification.</param>
-        public abstract void UpdateScale(Transform source, Transform target);
+        public virtual void UpdateScale(Transform source, Transform target)
+        {
+            if (isActiveAndEnabled && ValidCache(source, target))
+            {
+                DoUpdateScale(source, target);
+            }
+        }
+
+        /// <summary>
+        /// Updates the source position based on the target position.
+        /// </summary>
+        /// <param name="source">The source to modify.</param>
+        /// <param name="target">The target to utilize in the modification.</param>
+        protected abstract void DoUpdatePosition(Transform source, Transform target);
+        /// Updates the source rotation based on the target rotation.
+        /// </summary>
+        /// <param name="source">The source to modify.</param>
+        /// <param name="target">The target to utilize in the modification.</param>
+        protected abstract void DoUpdateRotation(Transform source, Transform target);
+        /// Updates the source scale based on the target scale.
+        /// </summary>
+        /// <param name="source">The source to modify.</param>
+        /// <param name="target">The target to utilize in the modification.</param>
+        protected abstract void DoUpdateScale(Transform source, Transform target);
+
+        /// <summary>
+        /// Caches the given source <see cref="Transform"/> and target <see cref="Transform"/> and determines if both are not null.
+        /// </summary>
+        /// <param name="source">The source to modify.</param>
+        /// <param name="target">The target to utilize in the modification.</param>
+        /// <returns><see langword="true"/> if the cache contains a valid source and target.</returns>
+        protected virtual bool ValidCache(Transform source, Transform target)

```

Comment:
Methods should be verbs, so `IsValidCache`? Though it additionally **sets**, too... 🤔 Maybe `ValidateCache`?